app/src/main/res/layout/activity_main.xml

```diff
@@ -6,6 +6,27 @@
     android:layout_height="match_parent"
     tools:context=".presentation.ui.MainActivity">
 
+    <com.google.android.material.appbar.AppBarLayout
+        android:id="@+id/appBarLayout"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:paddingTop="@dimen/spacing_default"
+        android:paddingBottom="@dimen/spacing_default"
```

Comment:
How about remove all paddings and make this thing more default? With standardised size and so one